.gitignore

```diff
@@ -133,4 +133,7 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-.DS_Store
\ No newline at end of file
+docker-compose.yml
```

Comment:
why is this in the `.gitignore`? We are not modifying it automatically as part of the script, so I don't see any reason why this should be excluded.

Reply:
My thought that we might want to exclude it would be to prevent accidentally committing changes when we modify this file for testing, but since that is a separate idea from excluding the dev file because we are modifying it with the script, it seems unrelated from this PR.

Reply:
Removed docker-compose.yml from .gitignore file.